great_expectations/checkpoint/checkpoint.py

```diff
@@ -156,6 +156,8 @@ def run(
             self._add_to_store()
 
         run_id = run_id or RunIdentifier(run_time=dt.datetime.now(dt.timezone.utc))
+        # TODO: Currently not providing checkpoint_id to result - is that okay?
```

Comment:
Note to self: this comment is out of date. we're passing it into `validation_definition.run` below.